doc/source/reference/api/api.rst

```diff
@@ -938,3 +938,31 @@ These are yt-provided functions:
    ~yt.testing.amrspace
    ~yt.testing.fake_random_ds
    ~yt.testing.expand_keywords
+
+These are for the pytest infrastructure:
+
+.. autosummary::
+
+    ~conftest.tempdir
+    ~conftest.answer_file
+    ~conftest.hashing
```

Comment:
Do these show up in the built API docs? You can see the built docs as a build artifact of the docs test job on jenkins.

If not you might need to move them into the yt module and then import them in conftest.py.

Reply:
They show up, but the functions from conftest.py can't actually be interacted with (that is, they are not links). I'll look into fixing it.